test/fit_tests/deploy/rackhd_source_install.py

```diff
@@ -184,56 +279,30 @@ def test05_install_rackhd_config_files(self):
-                                                  "echo 'netmask 255.255.252.0' >> /etc/network/interfaces.d/control.cfg"
-                                                  )['exitcode'], 0, "Network config failure.")
-        # If PDU network is present, configure
-        try:
-            IFLIST[8]
-        except IndexError:
-            print "**** No PDU network will be configured"
-        else:
-            self.assertEqual(fit_common.remote_shell("echo 'auto " + IFLIST[8] + "' > /etc/network/interfaces.d/pdudirect.cfg;"
-                                                      "echo 'iface " + IFLIST[8] + " inet static' >> /etc/network/interfaces.d/pdudirect.cfg;"
-                                                      "echo 'address 192.168.1.1' >> /etc/network/interfaces.d/pdudirect.cfg;"
-                                                      "echo 'netmask 255.255.255.0' >> /etc/network/interfaces.d/pdudirect.cfg"
-                                                      )['exitcode'], 0, "Network config failure.")
-
-    def test07_reboot_and_check(self):
-        print "**** Reboot and check installation."
-        # create startup files
-        self.assertEqual(fit_common.remote_shell(
-            "touch /etc/default/on-dhcp-proxy /etc/default/on-http /etc/default/on-tftp /etc/default/on-syslog /etc/default/on-taskgraph"
-            )['exitcode'], 0, "Install failure.")
-        # reboot
-        print "**** Rebooting appliance..."
-        fit_common.remote_shell("shutdown -r now")
-        print "**** Waiting for login..."
-        fit_common.countdown(30)
-        shell_data = 0
-        for dummy in range(0, 30):
-            shell_data = fit_common.remote_shell("pwd")
-            if shell_data['exitcode'] == 0:
-                break
+    def test06_startup(self):
+        print "Start services."
+        startup = open('startup.sh', 'w')
+        startup.write('cd ~/;nf start&\n')
+        startup.close()
+        fit_common.scp_file_to_ora('startup.sh')
+        self.assertEqual(fit_common.remote_shell("chmod 777 startup.sh;/etc/init.d/isc-dhcp-server restart")['exitcode'], 0, "dhcp startup failure.")
+        self.assertEqual(fit_common.remote_shell("nohup ./startup.sh")['exitcode'], 0, "RackHD startup failure.")
+        print "**** Check installation."
+        for dummy in range(0, 10):
+            try:
+                fit_common.rackhdapi("/api/2.0/config")
+            except:
+                fit_common.time.sleep(10)
             else:
-                fit_common.time.sleep(5)
-        self.assertEqual(shell_data['exitcode'], 0, "Shell test failed after appliance reboot")
-        fit_common.time.sleep(10)
+                break
         self.assertEqual(fit_common.rackhdapi("/api/2.0/config")['status'], 200, "Unable to contact RackHD.")
 
 if __name__ == '__main__':
-    fit_common.unittest.main()
\ No newline at end of file
+    fit_common.unittest.main()
```

Comment:
Same comment here.  This should just be unittest.main().   Reader assumes this is overridden.